modules/ROOT/pages/configure-cidr-range.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Pod CIDR Range for Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To configure the pod Classless Inter-Domain Routing (CIDR) range on an existing Runtime Fabric cluster:
+
+. Create a resource definition using a new `podCIDR` value. In the following example, the yaml file is named `pod-cidr.yaml` and `x.x.x.x/16` represents the new pod CIDR range:
++
+```
+kind: clusterconfiguration
+version: v1
+spec:
+  global:
+    podCIDR: x.x.x.x/16
+```
+. Run the following gravity command:
++
+```
+sudo gravity resource create pod-cidr.yaml --confirm
+```
+. Monitor the update and wait for the pod CIDR change to be configured on all cluster nodes. 
+Note: This operation causes a brief disruption to existing pods and redeploys each node as the 
```

Comment:
This does not look like the standard format for a Note notice?
I also wonder if we should warn users before engaging in the task, just in case they cannot afford the brief disruption?